cypress/support/step_definitions/annual-review/scheduler/an_email_for_the_string_key_date_is_sent_to_eligible_providers.js

```diff
@@ -0,0 +1,48 @@
+Then("an email for the {string} key date is sent to eligible providers", async function (keyDate) {
+  let list, audits;
```

Comment:
Maybe something for the CYB code conventions but it would be good to know people's opinions on:
```js
let list, audits;
```

vs

```js
let list;
let audits;
```
I personally prefer the latter but happy to do what is best for the team.

Reply:
Or even better, changing line **11** from:

```js
list = result;
```

to

```js
const list = result;
```

And a similar thing on line **41**.

But it's not that big of a deal 🤷

Reply:
aha, there maybe a better way to do this but `audits` and `list` are both needed in the last block and assigned to the results in subsequent `then`s.